content/basics/templates/resource-return-values/index.md

```diff
@@ -152,17 +174,47 @@ Outputs:
     Description: ID of the security group created
     Value: !GetAtt InstanceSecurityGroup.GroupId
 ```
-:::
+:::::tabs{variant="container"}
+::::tab{id="cloud9" label="Cloud9"}
+1. In the **Cloud9 terminal** navigate to `code/workspace/resource-return-values`:
+  :::code{language=shell showLineNumbers=false showCopyAction=false}
+  cd cfn101-workshop/code/workspace/resource-return-values
+  :::
+1. Use the AWS CLI to create the stack. The required parameter `--template-body` have been pre-filled for you.
+  :::code{language=shell showLineNumbers=false showCopyAction=false}
```

Comment:
```suggestion
  :::code{language=shell showLineNumbers=false showCopyAction=true}
```

Reply:
I think we want to keep it the flag as false as it overlaps with text. Part of initial review comments

Reply:
Since these are commands that need to be copied, it should be better to have the copy button even if it overlaps with the text. Maybe we could make the commands multiline if it overlaps